e2e/tests/e2e_happy_path/HappyPath.spec.ts

```diff
@@ -204,40 +208,41 @@ suite('Display source code changes in the running application', async () => {
 
         await warningDialog.waitAndCloseIfAppear();
     });
-});
+ });
 
 suite('Validation of debug functionality', async () => {
     test('Open file and activate breakpoint', async () => {
-        await projectTree.expandPathAndOpenFile(pathToJavaFolder, javaFileName);
+        await projectTree.expandPathAndOpenFileInAssociatedWorkspace(pathToJavaFolder, javaFileName);
         await editor.selectTab(javaFileName);
         await editor.moveCursorToLineAndChar(javaFileName, 34, 1);
         await editor.activateBreakpoint(javaFileName, 32);
     });
 
     test('Launch debug', async () => {
         await runTask('che: run-debug');
-
-        await ide.waitNotificationAndConfirm('A new process is now listening on port 8080', 120000);
-        await ide.waitNotificationAndOpenLink('Redirect is now enabled on port 8080', 120000);
+        await ide.waitNotificationAndConfirm('A new process is now listening on port 8080', 180000);
+        await ide.waitNotificationAndOpenLink('Redirect is now enabled on port 8080', 180000);
     });
 
     test('Check content of the launched application', async () => {
-        await previewWidget.waitContentAvailable(SpringAppLocators.springErrorMessageLocator, 60000, 10000);
+        await previewWidget.waitContentAvailableInAssoziatedWorkspace(SpringAppLocators.springTitleLocator, 60000, 10000);
+        await checkErrorMessInApplicationController();
     });
 
     test('Open debug configuration file', async () => {
-        await topMenu.selectOption('Debug', 'Open Configurations');
+        try {await topMenu.selectOption('Debug', 'Open Configurations'); } catch (e) {
```

Comment:
Could you place try-catch logic inside the dedicated method? It looks more clear.

Reply:
moved to separated method